hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -128,7 +130,21 @@ def _strategy_for(param: inspect.Parameter) -> Union[st.SearchStrategy, InferTyp
 def _get_params(func: Callable) -> Dict[str, inspect.Parameter]:
     """Get non-vararg parameters of `func` as an ordered dict."""
     var_param_kinds = (inspect.Parameter.VAR_POSITIONAL, inspect.Parameter.VAR_KEYWORD)
-    params = list(inspect.signature(func).parameters.values())
+    try:
+        params = list(inspect.signature(func).parameters.values())
+    except Exception:
+        # `inspect.signature` doesn't work on ufunc objects, but we can work out
+        # what the required parameters would look like if it did.
+        if not _is_probably_ufunc(func):
+            raise
```

Comment:
why not raise the exception?

```suggestion
    except Exception as e:
        # `inspect.signature` doesn't work on ufunc objects, but we can work out
        # what the required parameters would look like if it did.
        if not _is_probably_ufunc(func):
            raise e
```

Reply:
Bare `raise` has slightly different effects on the stack trace, and I like the simpler version more.